tox.ini

```diff
@@ -1,35 +1,69 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [tox]
+minversion = 3.18
 envlist =
-    py{27,py,35,36,37,38}{,-subunit},coverage,docs
+    lint
+    py27
+    py35
+    py36
+    py37
+    py38
+    py39
+    pypy
+    pypy3
+    coverage
```

Comment:
Yeah this drops docs and all the subunit variants, not good.

Reply:
Semi-fixed, but I'm now enumerating all the supported Python versions in .meta.toml, in `additional-envlist`.